tools/Experiments/src/ViewModels/QuietBackgroundProcessesViewModel.cs

```diff
@@ -0,0 +1,218 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.Linq;
+using System.Security.Principal;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using DevHome.Common.Helpers;
+using Microsoft.UI.Xaml;
+using Windows.UI.Xaml;
+
+namespace DevHome.Experiments.ViewModels;
+public class QuietBackgroundProcessesViewModel : INotifyPropertyChanged
+{
+    private readonly TimeSpan _zero;
+    private readonly bool _isElevated;
+    private readonly bool _validOsVersion;
+
+    public QuietBackgroundProcessesViewModel()
+    {
+        _zero = new TimeSpan(0, 0, 0);
+
+        var osVersion = Environment.OSVersion;
+        _validOsVersion = osVersion.Version.Build >= 26024;

```

Comment:
Is comparing against the build number the only way to tell if this feature is on the computer?